File: JavaAssignment5/src/main/java/Palindrome.java
public class Palindrome {

    Boolean palindromeChecker(String input){
        if(input==null) return null;
        String[] splitInputList = input.split("");
        for (int i=0; i< splitInputList.length; i++){
            if(!splitInputList[i].equals(splitInputList[splitInputList.length-(i+1)])) return false;
        }
        return true;
    }
}

File: Assignment 5/TypscriptAssignment5/src/Palindrome.ts
export class Palindrome{
    palindromeChecker(input: string): Boolean{
        if(input==null) return null;
        var splitInputList = input.split("");
        for (var i=0; i< splitInputList.length; i++){
            if(splitInputList[i] !== splitInputList[splitInputList.length-(i+1)]) return false;
        }
        return true;
    }
}